addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22486,16 +22486,19 @@ msgctxt "#39153"
 msgid "Windowing system:"
 msgstr ""
 
+#. Label for section of settings
```

Comment:
Sorry, this will not help the translators. You need to describe **what** the setting does.